posts/2025/farmOS-3.4.0/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: farmOS 3.4.0
```

Comment:
Wondering if it makes sense to pull 3.4.1 into the title to give it more weight since it's the actual recommended version now...?

```suggestion
title: farmOS 3.4.0 & 3.4.1
```

Reply:
Good point. What if instead of getting more specific we got less specific... What if we change it to "farmOS 3.4", and use that pattern in the blog posts and newsletters moving forward, so it is focused on the minor release number (aka new features), but omits the patch number. That is the intent of the blog posts and newsletters generally. We could probably remove the whole big about 3.4.1 too.

Reply:
Drafted a commit to this effect for consideration. Requesting re-review @symbioquine.

https://github.com/farmOS/farmOS-community-blog/pull/30/commits/82908b3a0eaaeee7385790cea600dbb77c71a071